tests/e2e_test.py

```diff
@@ -0,0 +1,230 @@
+# pylint: skip-file
```

Comment:
I think we should hold off on checking this in for now (but definitely still keep it around in your fork!!).

at some point we'll need to write up serious e2e testing for _all_ workloads, so we'll have this functionality then anyways

Reply:
(also removing this should fix the remaining lint issue)

Reply:
sounds good, deleted